qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -70,6 +70,12 @@ class BaseSpecification extends Specification {
 
     public static String coreImageIntegrationId = null
 
+    private static synchronizedGlobalSetup() {
```

Comment:
why not?
```suggestion
    private static synchronized synchronizedGlobalSetup() {
```

Reply:
Codenarc does not like it due to "SynchronizedMethod Rule".